src/ui/components/fiber-tree/ScrollSelectedIntoViewIfNeeded.tsx

```diff
@@ -1,16 +1,18 @@
 import * as React from "react";
-import { useSelectedId } from "../../utils/selection";
+import { useSelectedId, useHighlightedId } from "../../utils/selection";
 import { getBoundingRect, getOverflowParent } from "../../utils/layout";
 import { useTreeViewSettingsContext } from "./contexts";
 
 export const ScrollSelectedIntoViewIfNeeded = () => {
   const { selectedId } = useSelectedId();
+  const { highlightedId } = useHighlightedId();
   const { getFiberElement } = useTreeViewSettingsContext();
   const { groupByParent, showUnmounted } = useTreeViewSettingsContext();
 
   React.useEffect(() => {
+    const id = selectedId || highlightedId;
```

Comment:
Does `selectedId` should goes first, not a `hightlightedId`?

Reply:
Fixed